src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/AbstractCaseData.java

```diff
@@ -101,6 +101,11 @@ public class AbstractCaseData implements Serializable {
     @Column(name = "completed")
     private boolean completed;
 
+    @Setter
+    @Getter
+    @Column(name = "date_completed")
+    private LocalDateTime dateCompleted;
```

Comment:
I'd prefer for this to replace the 'completed' flag eventually. Not blocking this but could we have have a ticket created to discuss this and any further actions? 🙂

Reply:
created [HOCS-6410](https://collaboration.homeoffice.gov.uk/jira/browse/HOCS-6410)